### PR TITLE
LIT6823 to 25 and changes in others

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New records LIT6823-25 (Abb 156 related) and minor changes in some other records (3183 added structure).
https://github.com/BetaMasaheft/Documentation/issues/2356
https://github.com/BetaMasaheft/Documentation/issues/2353
https://github.com/BetaMasaheft/Documentation/issues/2354

### CODE REVIEW

File: 3001-4000/LIT3183Commen.xml [hunk 1]
```diff
@@ -65,6 +58,55 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <relation name="saws:isCommentOn" active="LIT3183Commen" passive="LIT1315Dodeka"/>
             </listRelation>
          </div>
+         <div type="edition">
+            
+            <div type="textpart" subtype="section" xml:lang="gez" xml:id="Hosea">
+               <label>On Genesis</label>
```
Comment (thea-m) [inline 3001-4000/LIT3183Commen.xml:64]: Genesis or Hosea? :) Also below, a few textparts have different names in the xml:id and labels

**Review by thea-m — CHANGES_REQUESTED** [review]

File: 3001-4000/LIT3183Commen.xml [hunk 2]
```diff
@@ -51,6 +43,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="PL" when="2016-07-26">Created XML record from Ethio authority google spreadsheet</change>
          <change when="2017-03-09" who="MV">Added title, keyword</change>
          <change who="DR" when="2017-03-31">Added relation</change>
+         <change who="DN" when="2023-05-12">Developped structure of the work</change>
```
Comment (eu-genia) [inline 3001-4000/LIT3183Commen.xml:46]: developed (one p)

File: new/LIT6823StoryMelchizedek.xml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2023-05-12">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT6823StoryMelchizedek" passive="BNFabb156"/>
```
Comment (eu-genia) [inline new/LIT6823StoryMelchizedek.xml:57]: is about PRS6988Melchize?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: 3001-4000/LIT3183Commen.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply 3001-4000/LIT3183Commen.xml:64]: Corrected.

**Review by thea-m — APPROVED** [review]

**Pull request merged**